practice_question7.rb

```diff
@@ -0,0 +1,10 @@
+File.open("friends.txt", "r") do |file|
+  for line in file.readlines()
+    puts line
+  end
+end
+file = File.open("friends.txt", "a")
+file.write("krishna raghuwanshi \n")
+file.close()
```

Comment:
Do we need to pass empty brackets?
```suggestion
file.close
```